integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -22,11 +22,12 @@ export class OpfResourceLoaderService {
   protected document = inject(DOCUMENT);
   protected platformId = inject(PLATFORM_ID);
 
-  protected readonly OPF_RESOURCE_ATTRIBUTE_KEY = 'data-opf-resource';
   protected readonly CORS_DEFAULT_VALUE = 'anonymous';
+  protected readonly OPF_RESOURCE_LOAD_ONCE_ATTR_KEY = 'opf-load-once';
+  protected readonly OPF_RESOURCE_ATTRIBUTE_KEY = 'data-opf-resource';
 
   protected embedStyles(embedOptions: {
-    attributes?: OpfKeyValueMap[];
+    attributes?: { [key: string]: string };
```

Comment:
Is it new type different than `OpfKeyValueMap[]`?

Reply:
Yes,
OpfKeyValueMap is {key:string,value:string}, we receive the list as input.
We perform a convertion to [key: string]: string to make handling easier and have same type as attribute in ScriptLoader service